langfuse/api/__init__.py

```diff
@@ -3,6 +3,9 @@
 from .resources import (
     AccessDeniedError,
     BaseEvent,
+    BasePrompt,
```

Comment:
Can you generate fern again for your changes? As discussed, i fixed something there. Important: before generating, please merge main again in your langfuse/langfuse feature branch to make that work.